### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rinkledink**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ suspend fun Payment.awaitSuccess(): Unit = throw RuntimeException("Payment Faile
 
 suspend fun main() {
   saga {
-    val order = saga { createOrder() }.compensate(::deleteOrder).bind()
-    val payment = saga { createPayment(order) }.compensate(::deletePayment).bind()
+    val order = saga(action = { createOrder() }, compensation = { ::deleteOrder })
+    val payment = saga(action = { createPayment(order) }, compensation = { ::deletePayment) })
```
Comment (nomisRev) [inline README.md:53]: Is this syntax broken? 😱 I should've set up compile time checked docs ><

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ repositories {
 }
 
 dependencies {
-    implementation("io.github.nomisrev:saga:0.1.3")
+    implementation("io.github.nomisrev:Saga-jvm:1.0.1")
```
Comment (nomisRev) [inline README.md:15]: This should also not be necessary :/ So something is broken in my release process.

**Review by nomisRev — APPROVED** [review]
Thank you for the PR @rinkledink 🙏 Your fixes indicate some issue with the API, and release process 😅 Let's merge it for now, but the issue should be looked into.

This functionality is moving over time to `arrow-fx-resillience` btw which will be a Kotlin alternative to Resilience4J.

File: README.md (hunk 2 above)
Comment (rinkledink) [reply README.md:15]: I picked up the dependency from here
https://mvnrepository.com/artifact/io.github.nomisrev/Saga-jvm
but now you mention that, I also see it here
https://mvnrepository.com/artifact/io.github.nomisrev/Saga
and here
https://mvnrepository.com/artifact/io.github.nomisrev/saga

File: README.md (hunk 1 above)
Comment (rinkledink) [reply README.md:53]: When using version 0.1.3 the syntax was as documented but when I upgraded to 1.0.1 it changed to
`  @SagaDSLMarker
  public suspend fun <A> saga(
    action: suspend SagaActionStep.() -> A,
    compensation: suspend (A) -> Unit,
  ): A`

**Comment by rinkledink** [comment]
I'm not sure I have permission to merge

File: README.md (hunk 2 above)
Comment (nomisRev) [reply README.md:15]: Sorry for my slow replies @rinkledink. Could you check if `implementation("io.github.nomisrev:Saga:1.0.1")` resolves correctly for you?

Comment (rinkledink) [reply README.md:15]: yeah, it resolves

**Comment by nomisRev** [comment]
Thanks for the update @rinkledink 🙌

**Pull request merged**